playbooks/edx_sandbox.yml

```diff
@@ -33,6 +33,8 @@
   roles:
     - role: swapfile
       SWAPFILE_SIZE: 4GB
+    - role: aws
```

Comment:
There is an unconditional inclusion of the `aws` role below. That needs to be removed in favour of this. The corresponding secure configuration repository has to be updated to set the `COMMON_ENABLE_AWS_INTEGRATION` variable to `True`.